### ISSUE TITLE
Crashing on start-up while "refreshing projects..."

### ISSUE THREAD

**Issue by atomworks** [bdkjones/CodeKit-1-Old#422]
Build: 6708

Got a problem where CodeKit is crashing every time I start it up. It spends a minute or so refreshing projects, hangs on 2 of 2 and when it finally starts going through the files it gets part way through and crashes.

I've removed all of the files from both of the active projects and no frameworks are being used. When the crash report comes up on re-launch and I hit open reports folder, it's empty!

What's the next step? It's frustrating as all my projects I've become dependant on CodeKit, now I need to re-work it all to get around this whilst it's sorted. Damn you CodeKit for being so insanely useful and the best workflow improvement I've gotten in ages!

Cheers!

**Comment by atomworks** [comment]
I've temporarily renamed my CodeKit3.storedata file so it's created a new one though I'm still looking for a proper fix as I've got 10 different projects that I'll need to use, all with varying settings.

**Comment by bdkjones** [comment]
Unfortunately, it sounds like the model file was corrupted somehow, so I don't think it will be possible to use the old store file. Sorry for the trouble. Can you recall what you did JUST before the crashes started?

-Bryan

**Comment by atomworks** [comment]
Ah damn, that's a shame.

I can in fact remember exactly what I was doing just before it broke! I had a project in an SVN of a live site I was working on, the live site however had changes on it made outside of the SVN so my plan was to basically pull down all the files off the server and then just commit whatever had changed. This didn't go that smoothly though... the details get a bit hazy but it went something along the lines of rename original folder, drop in the one from the live server, realise that wouldn't work as it didn't have the .svn folder. So, deleted the downloaded version of the folder, renamed the original one back into place (making a copy just incase) and merged the files a folder at a time (as OSX doesn't merge nicely). Somewhere in that was a couple of variations of the same thing that didn't work and probably just confused the hell out something.

Friday afternoon messy fix, I know... when I fired CodeKit back up after that the problems started. I noticed the .storedata file is XML... would there be an easy way for me to pull out potentially corrupt file and see if it'll fire up? 1 project setting to rebuild is better then 10. ;)
